docs/quick-start.md

```diff
@@ -210,93 +210,167 @@ kubectl exec -n vm "$VMSINGLE_POD_NAME" -- ls -l  /victoria-metrics-data
 # drwxr-xr-x    3 root     root          4096 Apr 30 12:20 tmp
 ```
 
-### Scraping
-
-#### VMAgent
+While you can push metrics directly as shown above, in most cases you won't need to.
+VictoriaMetrics can discover your applications and scrape their metrics. Let's set scrapping up next.
 
-Now let's deploy [`vmagent`](https://docs.victoriametrics.com/operator/resources/vmagent) resource.
+### Scraping
 
-Create file `vmagent.yaml` 
+The [VMAgent](https://docs.victoriametrics.com/operator/resources/vmagent/) discovers and scrape metrics from your apps.
+It uses special resources like [VMServiceScrape](https://docs.victoriametrics.com/operator/resources/vmservicescrape/) to know where to look.
+Once you create a scrape resource, `VMAgent` picks it up automatically and starts pulling metrics.
 
-```shell
-code vmagent.yaml
-```
-
-with the following content:
-
-```yaml
+Let’s start by creating a `VMAgent` manifest file:
+```bash
+cat <<EOF > vmagent-demo.yaml
 apiVersion: operator.victoriametrics.com/v1beta1
 kind: VMAgent
 metadata:
   name: demo
+  namespace: vm
 spec:
   selectAllByDefault: true
   remoteWrite:
-    - url: "http://vminsert-demo.vm.svc:8480/insert/0/prometheus/api/v1/write"
+    - url: "http://vmsingle-demo.vm.svc:8429/api/v1/write"
 ```
+The `selectAllByDefault` setting tells `VMAgent` to look for scrape resources in every namespace.
+The `remoteWrite.url` setting tells `VMAgent` where to send the scraped metrics.
+We’ll use the `VMSingle` storage we created earlier as the remote write destination.
 
-After that you can deploy `vmagent` resource to the kubernetes cluster:
-
-```shell
-kubectl apply -f vmagent.yaml -n vm
+Apply the manifest to your cluster:
+```bash
+kubectl -n vm apply -f vmagent-demo.yaml
 
 # vmagent.operator.victoriametrics.com/demo created
 ```
 
-Check that `vmagent` is running:
+To check if `VMAgent` is running, use:
+```bash
+kubectl get vmagent -n vm
 
-```shell
-kubectl get pods -n vm -l "app.kubernetes.io/instance=demo" -l "app.kubernetes.io/name=vmagent"
+# NAME   SHARDS COUNT   REPLICA COUNT   STATUS        AGE
+# demo                                  operational   41h
+
+kubectl get pods -n vm -l "app.kubernetes.io/name=vmagent"
 
 # NAME                            READY   STATUS    RESTARTS   AGE
-# vmagent-demo-6785f7d7b9-zpbv6   2/2     Running   0          72s
+# vmagent-demo-75bbfb6c5d-8htm8   2/2     Running   0          41h
 ```
 
-More information about `vmagent` resource you can find on 
-the [vmagent page](https://docs.victoriametrics.com/operator/resources/vmagent).
-
-#### VMServiceScrape
-
-Now we have the timeseries database (vmcluster) and the tool to collect metrics (vmagent) and send it to the database.
+Now let’s deploy a demo application that exposes some metrics. 
```

Comment:
I'd still recommend putting a link to the source code of this application, so power users can inspect it before deploying.